src/screens/auth/phone-auth/phone-auth-form.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import {
+  VStack,
+  Container,
+  Heading,
+  FormControl,
+  HStack,
+  Input,
+  Button,
+  useDisclose,
+  Text,
+  Pressable,
+  Menu,
+  ScrollView,
+} from 'native-base';
+import useLoginForm from './phone-auth-form-hook';
+import { AsyncError } from '../../../types';
+import COUNTRY_SELECT_OPTIONS from '../../../constants/countries';
+
+interface PhoneAuthFormProps {
+  onSuccess: () => void;
+  onError: (error: AsyncError) => void;
+}
+
+const PhoneAuthForm: React.FC<PhoneAuthFormProps> = ({
+  onSuccess,
+  onError,
+}) => {
+  const { formik, isSendOTPLoading } = useLoginForm({
+    onSendOTPSuccess: onSuccess,
+    onError: onError,
+  });
+
+  const { isOpen, onOpen, onClose } = useDisclose();
+
+  const handleSelectChange = (value: string) => {
+    const [countryCode, country] = value.split(', ');
+    formik.setFieldValue('countryCode', countryCode);
+    formik.setFieldValue('country', country);
+  };
+
+  const renderCountrySelectMenu = () => {
+    return (
+      <Menu
+        isOpen={isOpen}
+        onOpen={onOpen}
+        onClose={onClose}
+        trigger={triggerProps => (
+          <Pressable
+            {...triggerProps}
+            borderWidth={1}
+            justifyContent="center"
+            borderColor={'warmGray.300'}
+            rounded="sm"
+            px={2}
+          >
+            <Text>{`${formik.values.country} (${formik.values.countryCode})`}</Text>
+          </Pressable>
+        )}
+      >
+        <ScrollView maxH={'200px'}>
+          {COUNTRY_SELECT_OPTIONS.map(option => (
+            <Menu.Item
+              key={option.value}
+              onPress={() => {
+                handleSelectChange(option.value);
+                onClose();
+              }}
+            >
+              {option.label}
+            </Menu.Item>
+          ))}
+        </ScrollView>
+      </Menu>
+    );
+  };
+
+  return (
+    <VStack space={6}>
+      <Container>
+        <Heading
+          size="lg"
```

Comment:
Why is this wrapped around here so early in new line? It seems like a prettier issue. We should be able to atleast write code upto 80 or 100 characters in a line

Reply:
we can now write code upto 100 character in a line